eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -29,21 +29,6 @@
     }
   ],
   "include": [
-    {
-      "Agent": {
-          "ubuntu-24.04": { "OSVmImage": "env:LINUXVMIMAGE", "Pool": "env:LINUXPOOL" }
-      },
-      "JavaTestVersion": "1.21",
-      "AZURE_TEST_HTTP_CLIENTS": "netty",
-      "Options": {
-        "FromSource_SkipRebuild_Verify": {
-          "TestFromSource": true,
-          "RunAggregateReports": false,
-          "TestOptions": "-DskipCompile=true -DskipTestCompile=true -DcreateSourcesJar=false",
-          "TestGoals": "verify"
-        }
-      }
-    },
```

Comment:
I'm guessing this was disabled for a moment to run some validation but will be reverted, right?

Reply:
Purely for the dev feed release / pushing to get a repro quicker.